Winch/Util/ItemUtil.cs

```diff
@@ -25,17 +25,20 @@ internal static class ItemUtil
         { typeof(DamageItemData), new DamageItemDataConverter() },
     };
 
-    public static Dictionary<string, ItemData> HarvestableItemDataDict = new();
+    public static Dictionary<string, ItemData> harvestableItemDataDict = new();
+    public static Dictionary<string, ItemData> allItemDataDict = new();
 
```

Comment:
Meant to push this to a different branch 😅, but shouldn't cause any issues here.